Library/Formula/gwyddion.rb

```diff
@@ -23,6 +22,12 @@ class Gwyddion < Formula
   depends_on "gtksourceview" if build.with? "python"
 
   def install
+    inreplace "configure", 'PYTHON_LIBS=-l$libpython',
+                           %(py_prefix=`$PYTHON -c "import sys; print sys.prefix"`
+                           PYTHON_LIBS="-L${py_prefix}/lib -l$libpython")
+    inreplace "modules/pygwy/Makefile.in", '$(PYTHON_LIBS) $(PYGTK_LIBS) @GTK_LIBS@',
+                                           '-undefined dynamic_lookup $(PYGTK_LIBS) @GTK_LIBS@'
```

Comment:
Can you please submit these patches to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!